dspace/solr/qaevent/conf/solrconfig.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<!--
+    DSpace 'qaevent' core.
+
+    For more details about configurations options that may appear in
+    this file, see http://wiki.apache.org/solr/SolrConfigXml.
+-->
+<config>
+    <luceneMatchVersion>8.8.1</luceneMatchVersion>
+
```

Comment:
I think this is missing the `<lib>` tags which pull in the ICU analyzers that you use in `schema.xml`.  In `search` core they are these:
```
<!-- Include contributed libraries that we use in DSpace. -->
    <lib dir='${solr.install.dir}/contrib/analysis-extras/lib/'
         regex='icu4j-.*\.jar'/>
    <lib dir='${solr.install.dir}/contrib/analysis-extras/lucene-libs/'
         regex='lucene-analyzers-icu-.*\.jar'/>
```